app/grandchallenge/components/templates/components/partials/civ_inline.html

```diff
@@ -0,0 +1,30 @@
+{% load civ %}
+
+{% with interface=object.interface civ=object %}
+
+    {% if interface.super_kind == interface.SuperKind.VALUE %}
+        {# No fallback here as a value could be null #}
+        {% if interface.kind == interface.Kind.CHART %}
+            {% include "components/partials/civ_inline/value_chart.html" %}
+        {% elif civ|can_preview %}
+            {% include "components/partials/civ_inline/value_preview.html" %}
+        {% else %}
+            {% include "components/partials/civ_inline/value.html" %}
+        {% endif %}
+    {% elif interface.super_kind == interface.SuperKind.FILE %}
+        {% if not civ.file %}
+            {% include "components/partials/civ_inline/fallback.html" %}
+        {% elif civ|is_thumbnail_kind %}
+            {% include "components/partials/civ_inline/file_thumbnail.html" %}
+        {% else %}
+            {% include "components/partials/civ_inline/file.html" %}
+        {% endif %}
+    {% elif interface.super_kind == interface.SuperKind.IMAGE %}
+        {% if not civ.image %}
+            {% include "components/partials/civ_inline/fallback.html" %}
+        {% else %}
+            {% include "components/partials/civ_inline/image.html" %}
+        {% endif %}
+    {% endif %}
+
+{% endwith %}
```

Comment:
This seems to be duplicate logic, or there are some subtle differences that are hard to follow. Rather than having two files and a `{% block badge_content %}`, maybe it would be better to have one file with an if statement and to pass the variable `display_civ_inline=True/False`?

Reply:
Sorry, the PR overall looks good, just getting a bit lost as everything is spread out over multiple interlinked files.

Reply:
> Sorry, the PR overall looks good

Thanks ;-).

Now that I look at it with a bit more scrutiny I think I can push the small difference into the actual template.

I could have a single file with a variable, but then we'd lose the requested IDE compact since the base path would be dynamic of the templates. Furthermore, does that not swim into the 'do one thing well' philosophy?

Reply:
That's my point really, I don't think we need two templates for each interface type in different paths - but one template for each interface type with an if statement in there which would clearly show the differences between the two implementations.

Reply:
Ah, I see your point! Yes. That would put the two different renditions of a CIV nicely together.